ext/opcache/config.m4

```diff
@@ -68,41 +74,18 @@ if test "$PHP_OPCACHE" != "no"; then
       DASM_FLAGS="$DASM_FLAGS -D ZTS=1"
     fi
 
-    PKG_CHECK_MODULES([CAPSTONE], [capstone >= 3.0.0],
-        [have_capstone="yes"], [have_capstone="no"])
-    if test "$have_capstone" = "yes"; then
-        AC_DEFINE(HAVE_CAPSTONE, 1, [ ])
+    AS_IF([test x"$with_opcache_capstone" = "xyes"],[
```

Comment:
Do we need the x here?

Reply:
It can help with portability in some corner cases. I've never met one, but apparently some old buggy shells had problems comparing empty strings. Mainly I think it helps if `$with_opcache_capstone` begins with a dash. In that case, the "x" prevents it from looking like we're passing flags to `test`.

Reply:
I looked and there are a few select places where we already do this so I suppose it's ok (although likely useless as most test do not have it).